app/controllers/api/v1/npq_applications_controller.rb

```diff
@@ -68,6 +72,20 @@ def access_scope
       def npq_application
         @npq_application ||= npq_lead_provider.npq_applications.includes(:cohort, :npq_course, participant_identity: [:user]).find(params[:id])
       end
+
+      def funded_place
+        permitted_params.dig("attributes", "funded_place")
+      end
+
+      def permitted_params
```

Comment:
could we name this accepted permitted params to avoid confusion with other params for other actions?